index.md

```diff
@@ -178,36 +178,33 @@ FIXME
 
 ### Goals
 
-1.  Again, the easier it is for people to collaborate, the more likely they are to do so
-    (and to give you credit)
-2.  Two most reported barriers from [Steinmacher et al][steinmacher-newcomers]
-    are finding a task to start on
-    and problems setting up the local workspace to start work
-    *   So remove those
-3.  Additional disincentive is uncertainty: what am I allowed to do?
-    *   So remove that as well
-    *   Lack of an explicit license implies the author is keeping all rights
-        and others are not allowed to re-use
+You may work on your project with others, your known collaborators. But, even at an early stage, you may be interested to open your project so that you can attract new, as yet unknown collaborators. This section list approaches for making it straightforward for others to start collaborating on your project.
+
+1.  *Simplicity:* the easier it is for people to collaborate, the more likely they are to do so (and to give you credit)
```

Comment:
These aren't approaches, they're reasons - add an introductory sentence before the list to say that?